### PR TITLE
Fixing includes, adding missing TODO markers to koans, comments.

### PR DESCRIPTION
**Pull request by codecop**
Hi, thank you for the koans. A great exercise to get into C. While working through them I fixed a bunch of small things: Fixing includes, adding missing TODO markers to koans, formatting comments. Enjoy

### CODE REVIEW

File: src/about_control_statements.c [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #include "c_koans.h"
+#include <stdbool.h>
```
Comment (nbbeeken) [inline src/about_control_statements.c]: Unused include

File: src/about_io.c [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #include "c_koans.h"
-#include <criterion/criterion.h>
-#include <criterion/redirect.h>
 #include <stdio.h>
+#include <strings.h>
```
Comment (nbbeeken) [inline src/about_io.c]: Unused include

File: src/about_malloc.c [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #include "c_koans.h"
 #include <stdbool.h>
 #include <stdlib.h>
+#include <strings.h>
```
Comment (nbbeeken) [inline src/about_malloc.c]: Unused include

File: src/about_structs.c [hunk 4]
```diff
@@ -1,4 +1,5 @@
 #include "c_koans.h"
+#include <string.h>
```
Comment (nbbeeken) [inline src/about_structs.c]: Unused include

**Review by nbbeeken — CHANGES_REQUESTED** [review]
Hello :wave: thank you so much for your contribution! We really do appreciate you taking the time to share your fixes. I think everything is pretty good to go (especially some much needed fixes :rocket:) 
I caught a few includes that don't seem to be used listed below:

**Comment by codecop** [comment]
Curious. I added it because compiler was complaining. I will remove it of course.

**Comment by codecop** [comment]
Hi, the unused includes have been removed with the last (new) commit.

**Review by nbbeeken — APPROVED** [review]

**Comment by nbbeeken** [comment]
Thanks again for helping out 👍

**Pull request merged**